src/content/en/showcase/2016/iowa2016.md

```diff
@@ -0,0 +1,473 @@
+
+    <!-- Lazy pages manages the template stamping. It watches for route changes
+         and sets `template.if = true` on the appropriate template. -->
+    <lazy-pages>
+      <template is="dom-if" name="home">
+        <io-home-page date="2016-05-18T17:00:00Z"></io-home-page>
+      </template>
+
+      <template is="dom-if" name="schedule">
+        <io-schedule-page date="2016-05-18T17:00:00Z"></io-schedule-page>
+      </template>
+
+      <template is="dom-if" name="attend">
+        <io-attend-page></io-attend-page>
+      </template>
+    </lazy-pages>
+
+What I like about this is that every page is parsed and ready to go when the page loads but its CSS/HTML/JS only get executed on-demand (when its parent `<template>` is stamped). Dynamic + lazy views using web components FTW.
+
+**Future improvements**
+
+When the page first loads, we're loading all the [HTML Imports](http://www.html5rocks.com/en/tutorials/webcomponents/imports/) for each page at once. An obvious improvement would be to lazy load the element definitions only when they're needed. Polymer also has a nice helper for async loading HTML Imports:
+
+    Polymer.Base.importHref('io-home-page.html', (e) => { ... });
+
+IOWA doesn't do this because a) we got lazy and b) it's unclear just how much performance boost we would have seen. Our first paint was already ~1s.
+
+### Page lifecycle management
+
+The Custom Elements API defines "[lifecycle callbacks](https://w3c.github.io/webcomponents/spec/custom/#concept-custom-element-definition-lifecycle-callbacks)" for managing the state of a
+component. When you implement these methods, you get free hooks into the life
+of a component:
+
+    createdCallback() {
+      // automatically called when an instance of the element is created.
+    }
+
+    attachedCallback() {
+      // automatically called when the element is attached to the DOM.
+    }
+
+    detachedCallback() {
+      // automatically called when the element is removed from the DOM.
+    }
+
+    attributeChangedCallback() {
+      // automatically called when an HTML attribute changes.
+    }
+
+It was easy to leverage these callbacks in IOWA. Remember, every page is a self-contained DOM node. Navigating to a "new view" in our SPA is a matter of attaching a node to the DOM and removing another.
+
+We used the `attachedCallback` to perform setup work (init state, attach event listeners). When users navigate to a different page, the `detachedCallback` does cleanup (remove listeners, reset shared state). We also expanded the native lifecycle callbacks with several of our own:
+
+    onPageTransitionDone() {
+      // page transition animations are complete.
+    },
+
+    onSubpageTransitionDone() {
+      // sub nav/tab page transitions are complete.
+    }
+
+These were useful additions for delaying work and minimizing the jank between page transitions. More on this later.
+
+<h3 id="dry">DRYing up common functionality across pages</h3>
```

Comment:
You can use `### DRYing up common functionality across pages {: #dry }` if you want to use MD for titles, not a requirement but more consistent